.github/workflows/ci.yaml

```diff
@@ -38,6 +38,11 @@ jobs:
             php-version: '5.6'
             SKIP_LINT_TEST_CASES: 1
 
+          - operating-system: 'macos-latest'
+            php-version: '5.6'
```

Comment:
why?

Reply:
So we know it actually works on macos with php 5.6, just like we do for Windows.

When we drop PHP < 7.1, we can change both builds to 7.1.

Reply:
i believe we do not need to check OS-specific behaviour on many PHP envs. one is enough

Reply:
So we should replace the windows 5.6 and 7.3 builds by a single 7.4 build?

Reply:
Already in earlier pr